cmd/validate/image.go

```diff
@@ -453,6 +457,61 @@ func validateImageCmd(validate imageValidationFunc) *cobra.Command {
 				return err
 			}
 
+			if data.vsaEnabled {
```

Comment:
this is already a long function, i'd move this in its own function for readability 
```suggestion
			if data.vsaEnabled {
				generateVSA(...)
			}
```